### DISCUSSION TITLE
Knowledge Transfer Session (#1)

### DISCUSSION THREAD

**Discussion by mkanal** [eclipse-tractusx/item-relationship-service#386] [General]
|Session | Knowledge Transfer Session (1) | 
|----|----|
|  Date |  |  
|  Participants |  |  
|  Agenda |  |  
|  Recording |  <link to recording of session> |  
|  Ownership Q&A | <responsible person transferring the outcome to Q&A section> | 

# Preparation

In order to be prepared for a knowledge transfer session in the best possible way, the transition team SHOULD already think in advance about potential questions to be clarified in course of the sessions and to which points particular attention should be paid. For example, if it is a developer session, questions can already be asked on the basis of existing documentation or general questions about the technology stack should be prepared. 

# Questions
|Topic | Requestor | Participants | Question | Answer|    
|----|----|----|----|----|
|  ⁠Explain different scenarios with practical business use case examples, including workflows and involved parties. Explain concrete parts of results and their meaning. | @dsmf   |  |  |  |
|  |  |  |  |  |
|  |  |  |  |  |

# Notes
- [ ] 

# Open Points

# Critical Issues

## Debriefing

Each  team participant should reflect on a knowledge transfer session. This includes listing questions that came up afterward and including them in this  page until the questions are resolved in upcoming sessions. It is also useful to review and document the content of the session to ensure that important points have not been missed out.

**Questions documented within this KTS should be made available in the Q&A section located in the wiki.**

**Comment by dsmf** [comment]
Explain different scenarios with practical business use case examples including workflows and involved parties. Explain concrete parts of results and their meaning.

**Comment by dsmf** [comment]
In the meeting, we gathered specific ideas on how we can improve the documentation. @ds-jhartmann is creating the PBIs for these. We also created a new discussion for collecting further ideas on how to improve documentation: https://github.com/eclipse-tractusx/item-relationship-service/discussions/407

**Comment by mkanal** [comment]
Good moring @dsmf, @ds-alexander-bulgakov  @ds-jhartmann @ds-ext-kmassalski  @ds-psosnowski ,
as I see only one topic in the Discussion https://github.com/eclipse-tractusx/item-relationship-service/discussions/386 there might not be any need for Knowledge Transfer in the team. KT seems to be sufficient right now. Matthias, please organize a session with Krysztof or Jaro to get share knowledge. You could inform Pawel as well, and he is free to join this session. Do you agree? In this case, I would close the discussion?

**Comment by mkanal** [comment]
@dsmf please organize a KT session for this topic and afterward prepare discussion for the next KT Session. Thx.
